Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:dubnium-alpine
+
+WORKDIR /opt/openhim-mediator-file-queue
+
+COPY package.json npm-shrinkwrap.json ./
```

Is there a difference between `.` and `./` ?

```suggestion
COPY package.json npm-shrinkwrap.json .
```

---

@MattyJ007 `.` will not work when copying multiple files, that, at least, is the warning I got from VScode

---

Awesome